Add an option to disable the HTTPS security checks (#145)
megaease/easeprobe: README.md

    ca: /path/to/file.ca
    cert: /path/to/file.crt
    key: /path/to/file.key
    # TLS

>     insecure: true # skip the security check, use for self-signed certs. default: false

Reviewer: ```suggestion
    insecure: true # skip any security checks, useful for self-signed and expired certs. default: false
```
